### PR TITLE
Apply strict formatting for all compose files

### PR DESCRIPTION
**Pull request by louneskmt**
This PR applies a strict and fix formatting for all Umbrel and apps compose files. This was necessary as the previous formatting was very inconsistent and different across all files.

It needs careful reviews and testing to check everything still works and hasn't been impacted by one of these changes.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,188 +1,197 @@
 version: '3.7'
```
Comment (AaronDewes) [inline docker-compose.yml:1]:
```suggestion
version: "3.7"
```

This is using " in every app, so we should use it here too.

**Review by AaronDewes — APPROVED** [review]
Only one small change.

File: docker-compose.yml (hunk 1 above)
Comment (louneskmt) [reply docker-compose.yml:1]: Sure, I missed this one 😁

**Review by AaronDewes — APPROVED** [review]

**Comment by louneskmt** [comment]
@AaronDewes Have you been able to test this? Trying now.

**Comment by AaronDewes** [comment]
It started fine for me on a custom Ubuntu setup, but I haven't done much testing yet.

**Comment by louneskmt** [comment]
Ideally, we should try to install each app and test them all, to see if these changes didn't mess up with some app configuration.

**Pull request closed without merging**